src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -1,91 +1,87 @@
+    {% elif status === ClaimStatus.CLAIM_REJECTED %}
+      {% if claim.response.defenceType === DefenceType.ALREADY_PAID %}
+         {{ claimRejectedAlreadyPaidForClaimantDashboard(claim) }}
+        {% elif claim.response.freeMediation === FreeMediationOption.NO %}
+         {{ claimRejectedFreeMediationNOForClaimantDashboard(claim) }}
       {% else %}
-        {{ t('You’ve rejected the claim. The case will be reviewed by a judge and might go to court.' ) }}
+        {{ claimRejectedFreeMediationYESForClaimantDashboard(claim) }}
       {% endif %}
-    {% endif %}
-  {% elseif status === ClaimStatus.OFFER_ACCEPTED %}
-    {{ t('The claimant has accepted your offer. You need to sign a legal agreement to settle out of court.') }}
-  {% elseif status === ClaimStatus.OFFER_REJECTED %}
-    {{ t('{{ claimantName }} has rejected your offer.', { claimantName: claimantName }) | safe }}
-  {% elseif status === ClaimStatus.OFFER_SUBMITTED %}
-    {{ t('You made an offer to settle out of court.') }}
-  {% elseif status === ClaimStatus.MORE_TIME_REQUESTED %}
-    {{ t('You need to respond by {{ responseDeadline }}. You asked for an extra 14 days to respond.',
-    { responseDeadline: claim.responseDeadline | date }) }}
-  {% elseif status === ClaimStatus.NO_RESPONSE %}
-    <a href="{{ url }}">{{ t('Respond to claim') }}</a>
-    <span class="block mobile-inline">
-      {{ timeRemaining(claim.remainingDays, isAfter4pm) }}
-    </span>
+    {% elif status === ClaimStatus.OFFER_SUBMITTED %}
+      {{ offerSubmittedForClaimantDashboard(claim) }}
+    {% elif status === ClaimStatus.OFFER_REJECTED %}
+      {{ offerRejectedForClaimantDashboard(claim) }}
+    {% elif status === ClaimStatus.OFFER_ACCEPTED %}
+      {{ offerAcceptedForClaimantDashboard(claim) }}
+    {% elif status === ClaimStatus.OFFER_SETTLEMENT_REACHED %}
+      {{ offerSettlementReachedDashboard(claim) }}
   {% endif %}
 {% endmacro %}
 
-{% macro caseStatusForClaimant(status, claim, url) %}
-  {% set defendantName = claim.claimData.defendant.name %}
-  {% if status === ClaimStatus.CCJ_REQUESTED %}
-    {{ t('You requested a County Court Judgment on {{ ccjRequestedAt }}. We will contact you within 5 working days.',
-    { ccjRequestedAt: claim.countyCourtJudgmentRequestedAt | date }) }}
-  {% elseif status === ClaimStatus.ELIGIBLE_FOR_CCJ %}
-    <p>{{ t('You can request a County Court Judgment') }}</p>
-  {% elseif status === ClaimStatus.OFFER_SETTLEMENT_REACHED %}
-    {{ t('You both signed an agreement to settle out of court.') }}
-    <p>
-      <a href="{{ OfferPaths.agreementReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download the agreement') }}</a>
-    </p>
-  {% elseif status === ClaimStatus.CLAIM_REJECTED %}
+
+{% macro caseStatusForDefendant(status, claim, url, isAfter4pm) %}
+  {% if status === ClaimStatus.NO_RESPONSE %}
+      {{ noResponseForDefendantDashboard(claim, url) }}
+    {% elif status === ClaimStatus.MORE_TIME_REQUESTED %}
```

Comment:
all `elif` are on the same level of intend as the first `if`.

REQ. please reformat

Reply:
👍